### DISCUSSION TITLE
I deleted the internal partition via gparted, armbian on sdcard no longer boots?

### DISCUSSION THREAD

**Discussion by yntkts** [ophub/amlogic-s9xxx-armbian#2704] [Amlogic]
Hello folks,

I have a HG680-P device installed armbian on sdcard. It worked great until **I deleted the internal partition via gparted**.

Now when it boots it's just a black screen. is there any way to restore it, or boot armbian without depending on the internal partition (directly to sdcard)?

Thank you.

**Comment by ophub** [comment]
Flash an original Android system to eMMC and restore the original Android system.

**Reply by yntkts** [reply]
Thank you for your help.

I've tried upgrading via sdcard stuck upgrading approximately 1% forever. While using **amlogic USB burning tool** it can't continue at step 4/5, it says _Error romcode/initialize DDR_, precisely in the log says it didn't find `DDR_ENC.USB`, _Download DDR_ENC.USB,size: 0!_. Tried a few sdcards with no luck either.

![photo_2024-12-20_16-44-16](https://github.com/user-attachments/assets/82046a9e-7012-4b32-8ae7-552266e50bb0)

**Comment by yntkts** [comment]
Hi @ophub , after some weird experiments, I can skip the error. **It doesn't have to use original rom to restore**. I use atvXperience. Now the problem is sdcard can't boot armbian. What needs to be considered, or should be adjusted with the current rom (atvXperience)? Thanks.

_UPDATE:_
I assume I have to replace the internal `/dev/block/bootloader`? I have `dd` and root access,, btw, but I'm not sure which `bootloader.img` I should use?

_UPDATE:_
FIXED by myself. To boot to sdcard type `reboot update` in terminal emulator.

**Reply by tantos8** [reply]
Hei @yntkts, colud you make a detail what you mean with weird experiments, as i have almost same mistake/problem about partitionibg emmc with gparted
Mine cant boot at all, just blank, cant boot sdcard to.

**Reply by yntkts** [reply]
@tantos8 just try flash with some rom, mine atvXperience android 7. the only challenge is to _find_ the right state/moment (e.g. recovery mode / adb mode or something). if the android still boots, try `reboot update` (from terminal emulator), this will ensure to boot from sdcard first. usually it can also be flashed in this mode.

**Comment by 2hard4alot** [comment]
@yntkts Hi, there. I got the same problem here. I have hg680p with custom rom, and running Openwrt via usb. But I deleted internal partition (in hope for turning internal emmc to swap), but got bricked instead.
Tried flashing via usb burning tool and stuck in cant find DDR_ENC.USB error. tried flashing bootloader to sdcard and not working at all. I hope you can share your step by step solution. Thanks!
